CMakeOpenCLPythonPackage.txt

```diff
@@ -0,0 +1,113 @@
+
+set(BUILD_SHARED_LIBS OFF CACHE BOOL "" FORCE)
+set(BOOST_VERSION_DOT "1.73")
+string(REPLACE "." "_" BOOST_VERSION_UNDERSCORE ${BOOST_VERSION_DOT})
+
+set(OPENCL_HEADER_REPOSITORY "https://github.com/KhronosGroup/OpenCL-Headers.git")
+set(OPENCL_HEADER_TAG "3f7c1f77f8e1a3c9b48765c713ca19013e95b91e")
+
+# FIXME -- remove reference to OpenCL ICD Loader fork after decision on whether or not to include Linux build
```

Comment:
can you please remove `FIXME` / `TODO` entries and create issues for them? We have a strong preference for using issues to track work in this project instead of TODO comments

Reply:
This code was not intended for issue tracking.  Our primary goal is to build for Windows platforms, but I thought (as I mentioned in the overview above) that this technique might be useful for non-Windows platforms as well and that might be decided in this PR.

Since it hasn't generated any discussion, I've removed (1) the new code in this PR related to non-Windows builds (they won't complete successfully without an upstream change to the ICD Loader repo that hasn't happened yet), (2) the TODO comments (per your request), and (3) the reference to my ICD fork (obviously).  If desired, the non-Windows code can be added in a later PR.